_haikus/unimaginative_cloud.md

```diff
@@ -0,0 +1,8 @@
+--- layout: haiku
+title: Unimaginative Cloud
+author: pookzzz
+---
+
+Clouds float pass<br>
+Looks like dog, cat, jellyfish<br>
+Maybe just cloud<br>
```

Comment:
This line needs one more syllable. If you want to use a similar line, you can rewrite it to be:

```
Maybe just a cloud<br>
```